utils/cgn_checks.ts

```diff
@@ -97,7 +97,8 @@ export const toBirthDate = (fiscalCode: FiscalCode): Option<Date> => {
  * @param fiscalCode: the citizen's fiscalCode
  */
 export const extractCgnExpirationDate = (
-  fiscalCode: FiscalCode
+  fiscalCode: FiscalCode,
+  cgnUpperBoundAge: number = CGN_UPPER_BOUND_AGE
```

Comment:
What if this value is used as the default value inside the config?
`cgnUpperBoundAge` should become required.

Reply:
Yep but we should rename the env variables :)